revscoring/features/wikibase/datasources/excluded_qids.csv

```diff
@@ -0,0 +1,477 @@
+328,English Wikipedia
```

Comment:
This is Wikidata specific and probably shouldn't be included in a wikibase-general revscoring.  Maybe we should move this into wikiclass.